evm_loader/cli/src/errors.rs

```diff
@@ -55,43 +48,6 @@ pub enum NeonCliError {
     /// Incorrect program
     #[error("Incorrect program {0:?}")]
     IncorrectProgram(Pubkey),
-    /// Account not found at address
-    #[error("Account not found at address {0:?}.")]
```

Comment:
Is that really safe to remove these error codes from the public enum?

Reply:
Yes. And this enum is not public.
It's only visible inside the current crate.